lib/vagrant.rb

```diff
@@ -0,0 +1,17 @@
+module OS
```

Comment:
there is a ruby gem names os which doing the same. it is reinventing the weal?

Reply:
I have just looked for this, 
If I will try to use a missing gem, I would need to automatically install it.
(I assume I will need another extra gem for this)
When we will have an external wrapper, we will be able to install it externally.
I couldn't find other a proper way.

Reply:
then use a different approach:
https://stackoverflow.com/questions/26811089/vagrant-how-to-have-host-platform-specific-provisioning-steps

Reply:
LOL, this is exactly where I took my solution from 
but didn't looked down :(
fixed

Reply:
done